contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -0,0 +1,358 @@
+impl Felt252TryIntoBool of TryInto::<felt252, bool> {
+    fn try_into(self: felt252) -> Option<bool> {
+        if self == 0 {
+            return Option::Some(false);
+        }
+        if self == 1 {
+            return Option::Some(true);
+        }
+        Option::None(())
+    }
+}
+
+impl BoolIntoFelt252 of Into::<bool, felt252> {
+    fn into(self: bool) -> felt252 {
+        if (self) {
+            return 1;
+        }
+        return 0;
+    }
+}
+
+#[contract]
+mod SequencerUptimeFeed {
+    use super::Felt252TryIntoBool;
+    use super::BoolIntoFelt252;
+    use starknet::ContractAddress;
+    use starknet::StorageAccess;
+    use starknet::StorageBaseAddress;
+    use starknet::SyscallResult;
+    use starknet::storage_read_syscall;
+    use starknet::storage_write_syscall;
+    use starknet::storage_address_from_base_and_offset;
+
+    use box::BoxTrait;
+    use traits::Into;
+    use traits::TryInto;
+    use option::OptionTrait;
+
+    use chainlink::libraries::ownable::Ownable;
+    use chainlink::libraries::access_controller::AccessController;
+    use chainlink::libraries::simple_read_access_controller::SimpleReadAccessController;
+    use chainlink::libraries::simple_write_access_controller::SimpleWriteAccessController;
+    use chainlink::ocr2::aggregator::Round;
+    use chainlink::ocr2::aggregator::IAggregator;
+
+    #[derive(Copy, Drop, Serde, PartialEq)]
+    struct RoundTransmission {
```

Comment:
Note: I would've reused chainlink::ocr2::aggregator::Aggregator::Transmission but the answer type is a bool here

Reply:
Agreed, when we talked about it I forgot that this actually has to match the IAggregator interface. Since we're reading it into a bool, then immediately casting it to u128 to match the interface, we might as well just use the same types